Plots/Contours/NCL_lb_1.py

```diff
@@ -0,0 +1,93 @@
+'''
+NCL_lb_1.py
+===========
+This script illustrates teh following concepts:
+    - Making a horizontal colorbar
+    - Changing the colorbar labels
+    - Setting color maps using the new standard
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/lb_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/lb_1_lg.png
+'''
```

Comment:
```suggestion
"""
NCL_lb_1.py
===========
This script illustrates teh following concepts:
    - Making a horizontal colorbar
    - Changing the colorbar labels
    - Setting color maps using the new standard

See following URLs to see the reproduced NCL plot & script:
    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/lb_1.ncl
    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/lb_1_lg.png
"""
```

Make sure docstrings use """ instead of '''

Reply:
I realized I wrote "teh" instead of "the" but I changed that before fixing this so I just went in and did it manually!